hip_data_tools/google/common.py

```diff
@@ -0,0 +1,32 @@
+from attr import dataclass
+
+
+@dataclass
+class GoogleApiConnectionSettings:
+    """Encapsulates the Google API connection settings"""
+    key_file_path: str
+
+
+class GoogleApiConnectionManager:
+    """
+    Google API connection manager abstract class
+    """
+
+    def __init__(self, settings: GoogleApiConnectionSettings):
+        self.settings = settings
+        self.scope = ['https://spreadsheets.google.com/feeds',
+                      'https://www.googleapis.com/auth/drive']
+
+    def _credentials(self):
+        """
+        Get the credentials for a given google service
+        Returns (ServiceAccountCredentials): credentials object to authorize google services
```

Comment:
use the IntelliJ formatter before commiting

Reply:
Has used it already

Reply:
fix your intellij editor settings then 
![image](https://user-images.githubusercontent.com/21052816/74693099-60944080-523e-11ea-9704-ee5602242d0a.png)

Reply:
fixed